transport/server/opts.go

```diff
@@ -43,6 +46,22 @@ func WithHTTPMux(mux *chi.Mux) Option {
 	return server.WithHTTPMux(mux)
 }
 
+// WithHTTPRouterMux func sets HTTPMux Router
 func WithHTTPRouterMux(mux transport.Router) Option {
 	return server.WithHTTPRouterMux(mux)
 }
+
+// WithHTTPServer sets HTTP Server to use insted of the default
+func WithHTTPServer(srv *http.Server) Option {
+	return server.WithHTTPServer(srv)
+}
+
+// WithHTTPGracefull applies Gracefull shutdown func to server
+func WithHTTPGracefull(fn func(sc chan os.Signal) func() error) Option {
```

Comment:
`Gracefull` -> `Graceful`; also as I've already mentioned - it's not clear what can we use that for. Looks like a hack for your own code

Reply:
Huh. WithHTTPGracefull moved to WithGracefull in server/opts.go.
What is your way to gracefully shutdown HTTP server?
HTTP server is hidden in private variable. How to call HTTP server Shutdown func outside clay lib?

Reply:
`clay/server.Server.GracefulStop()` should be a way to gracefully stop HTTP and gRPC servers. Your `server.StopWithTimeout()` func does just that as I understand - what's the reason for another way to do the same thing?